sound/soc/sof/ipc4-priv.h

```diff
@@ -19,12 +19,17 @@
  * @num_fw_modules : Number of modules in base FW
  * @fw_modules: Array of base FW modules
  * @nhlt: NHLT table either from the BIOS or the topology manifest
+ * @max_fw_libs: Maximum number of FW libraries support by the FW including the base firmware
+ * @module_lib_info: Array of struct snd_sof_module_library_info containing information
+ *		     about all 3rd party modules
```

Comment:
so index0 is invalid then since it's not 3rd party modules?

Also the notion of 3rd party is a bit too vague, this could also be used for code that's part of the public SOF tree but not part of the base firmware.